public/app/features/alerting/alert_tab_ctrl.ts

```diff
@@ -87,6 +87,7 @@ export class AlertTabCtrl {
     switch (type) {
       case "email": return "fa fa-envelope";
       case "slack": return "fa fa-slack";
+      case "victorops": return "fa fa-exclamation-triangle";
```

Comment:
I don't think using `fa-exclamation-triangle` is a good idea as usually `fa-exclamation-triangle` signifies error. May be you can use something else. Nearest to what I found to VictorOps logo was `fa-pagelines`.